test/src/sw/redis++/async_test.h

```diff
@@ -199,6 +199,13 @@ void AsyncTest<RedisInstance>::_test_hash() {
                 this->set_ready();
             });
     _wait();
+
+    set_ready(false);
+    std::map<std::string, std::string> mkeys = {{test_key("str")+"1", "val1"}, {test_key("str")+"2", "val2"}};
```

Comment:
1. Use std::unordered_map instead of std::map, since this file doesn't include <map>, some compiler might fail to compile.
2. Register these two keys in KeyDeleter at the beginning of this method, so that they will be automatically deleted when the test is done.

```
auto key = test_key("set");
auto key1 = test_key("str1")
auto key2 = test_key("str2");

KeyDeleter<RedisInstance> deleter(_redis, {key, key1, key2});

// other code here

std::map<std::string, std::string> mkeys = {{key1, "val1"}, {key2, "val2"}}
```